voluptuous/schema_builder.py

```diff
@@ -854,11 +869,13 @@ def is_callable(key_):
     # Remove markers should match first (since invalid values will not
     # raise an Error, instead the validator will check if other schemas match
     # the same value).
-    priority = [(1, is_remove),  # Remove highest priority after values
-                (2, is_marker),  # then other Markers
-                (4, is_type),  # types/classes lowest before Extra
-                (3, is_callable),  # callables after markers
-                (5, is_extra)]  # Extra lowest priority
+    priority = [
+        (1, is_remove),  # Remove highest priority after values
+        (2, is_marker),  # then other Markers
+        (4, is_type),  # types/classes lowest before Extra
+        (3, is_callable),  # callables after markers
+        (5, is_extra),
+    ]  # Extra lowest priority
```

Comment:
move this comment back to the respective line

Reply:
sure thing.